.github/workflows/edr-ci.yml

```diff
@@ -157,6 +159,44 @@ jobs:
             **/edr-cache
           key: edr-ts-rpc-cache-v1-${{ matrix.os }}-${{ hashFiles('**/edr-cache/**/*') }}
 
+  test-edr-ts-integration:
```

Comment:
I renamed this job and moved it closer to the other test jobs to make it easier to cross-reference job description patterns